src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,36 @@
+package duke.task;
+
+import java.io.Serializable;
+
+public class Task implements Serializable {
+    protected String status;
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.status = " ";
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String toggleStatus() {
```

Comment:
Maybe toggleStatus could be made private (if it isnt referenced in other classes)?